tubesync/common/templates/base.html

```diff
@@ -64,6 +100,51 @@ <h1>TubeSync</h1>
       </div>
     </footer>
 
+    <script>
```

Comment:
This script is doing 2 things:

1. Sets up the dark mode toggle button in the header (default styles and click handling)
2. Sets up a click handler for the hamburger menu when the viewport is small